85.md

```diff
@@ -0,0 +1,85 @@
+NIP-85
+======
+
+Trusted Assertions
+------------------
+
+`draft` `optional`
+
+Certain calculations require access to a large volume of events, making it impossible to perform them directly on clients. This NIP allows service providers to handle such computations and generate signed events with results in real-time. Each calculation type is assigned an event kind and tag. Users can specify which service providers they trust, enabling clients to load and display the results based on those trusted sources.
+
+Assertions are always addressable events with the `d` tag pointing to the subject of the assertion. The following kinds and tags have been specified:
+
+| name                    | kind  | `d` tag value | result tags                         | 
+| ----------------------- | ----- | ------------- | ----------------------------------- |
+| Follower Count          | 30382 | pubkey        | `"followers"` (int)                 | 
+| User Rank               | 30382 | pubkey        | `"rank"` (int, normalized 0-100)    | 
```

Comment:
I think the relative rank should be made via a DVM, where there is a request/response that facilitates creating each pair that matters. Otherwise, generating ranks for every pairs of public keys in Nostr will take a huge processing power.